runtime/suggestion.js

```diff
@@ -0,0 +1,65 @@
+/**
+ * @license
+ * Copyright (c) 2017 Google Inc. All rights reserved.
+ * This code may only be used under the BSD style license found at
+ * http://polymer.github.io/LICENSE.txt
+ * Code distributed by Google as part of this project is also
+ * subject to an additional IP rights grant found at
+ * http://polymer.github.io/PATENTS.txt
+ */
+"use strict";
+
+const assert = require('assert');
+
+class Suggestion {
```

Comment:
I used addEventListener instead of onclick and createTextNode instead of innerHTML (is there a better way?).
This is an internal helper class now (not exported) and it encapsulates all the suggestion-related DOM work. I think there will be more to this class later, it'll probably be split similarly to Slot and SlotDom, so I think it is better outside of SuggestionComposer.
Or do you prefer I got rid of it for now and just used a plain object?